receiver/awsecscontainermetricsreceiver/awsecscontainermetrics/docker_stats.go

```diff
@@ -14,16 +14,21 @@
 
 package awsecscontainermetrics
 
+import "time"
+
 // ContainerStats defines the structure for container stats
 type ContainerStats struct {
-	Name string `json:"name"`
-	ID   string `json:"id"`
+	Name    string    `json:"name"`
+	ID      string    `json:"id"`
+	Read    time.Time `json:"read"`
+	PreRead time.Time `json:"preread"`
 
 	Memory      MemoryStats             `json:"memory_stats,omitempty"`
 	Disk        DiskStats               `json:"blkio_stats,omitempty"`
 	Network     map[string]NetworkStats `json:"networks,omitempty"`
 	NetworkRate NetworkRateStats        `json:"network_rate_stats,omitempty"`
 	CPU         CPUStats                `json:"cpu_stats,omitempty"`
+	PreCPU      CPUStats                `json:"precpu_stats,omitempty"`
```

Comment:
Let's use Previous instead of Pre in the code (not json which is fixed I suppose) for readability.

Reply:
Updated.